command/commands.go

```diff
@@ -210,6 +211,11 @@ func Commands(metaPtr *Meta, agentUi cli.Ui) map[string]cli.CommandFactory {
 				Meta: meta,
 			}, nil
 		},
+		docklog.PluginName: func() (cli.Command, error) {
```

Comment:
Yeah this should be possible. Since external plugins are forked and a separate binary, having the plugin author embed their own sub-command like this in their plugin would be supported.